bpf/join_ep.sh

```diff
@@ -28,6 +28,9 @@ function bpf_compile()
 	OUT=$2
 	TYPE=$3
 
+	clang -E -O2 -target bpf					\
```

Comment:
How long does this take? We will pay this cost every time we build endpoints. Can we make it dependant on the debug flag?

Reply:
Ack, making that change.